docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst

```diff
@@ -41,6 +41,33 @@ The following counters are available:
 
   decoder_error, Counter, Number of times a message wasn't decoded
   request_bytes, Counter, Number of bytes in decoded request messages
+  connect_rq_bytes, Counter, Number of bytes in decoded connect request messages
```

Comment:
I think these new fields also need the `enable_per_opcode_request_response_bytes_metrics` field to be set. If so, please update this doc to clarify which stats are enabled by which flag.